tests/drivers/rpc_flash/testcase.yaml

```diff
@@ -0,0 +1,9 @@
+common:
+  tags:
+    - drivers
+    - flash
+tests:
+  drivers.rpc_flash:
+    platform_allow: nrf5340dk_nrf5340_cpuapp
```

Comment:
incorrect platform name convention (currenlty the name should be compatibile with HWMv2)
```suggestion
    platform_allow: nrf5340dk/nrf5340/cpuapp
```

Reply:
Fixed. forgot to update to HWmod2